### DISCUSSION TITLE
News Settings Busted. How do I fix?

### DISCUSSION THREAD

**Discussion by joezimjs** [nextcloud/all-in-one#3007] [Questions] [answered]
At one point I was at mydomain.com/settings/admin/news and I think some autofill nonsense happened and filled the field for how often to check for updates. Now when I go to that settings page I get the following:

### Internal Server Error
The server was unable to complete your request. If this happens again, please send the technical details below to the server administrator. More details can be found the server log.

It then gives the technical details (remote address and request ID). So the questions:

1. How do I see the server log?
2. If I'm right about it being bad configuration, how can I change it if I can't access this page? Is it in a file or in the database? If in a file, which file? If in the database, how do I connect to the database and find that setting in it?

**Comment by szaimen** [comment, marked as answer]
> * How do I see the server log?

Navigate to `https://your-nc-domain/settings/admin/logging`

> * If I'm right about it being bad configuration, how can I change it if I can't access this page? Is it in a file or in the database? 

Most likely yes, but this is better asked at and reported to https://github.com/nextcloud/news/issues
